lib/api/controllers/securityController.js

```diff
@@ -733,7 +753,7 @@ function mDelete (kuzzle, type, request) {
  */
 function checkSearchPageLimit(request, limit) {
   if (request.input.args.size) {
-    let size = request.input.args.size - (request.input.args.from || 0);
+    const size = request.input.args.size - (request.input.args.from || 0);
 
     if (size > limit) {
       throw new SizeLimitError(`Search page size exceeds server configured documents limit (${limit})`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 104 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)